pastis/pastis_analysis.py

```diff
@@ -155,6 +167,8 @@ def full_modes_from_themselves(instrument, pmodes, datadir, sim_instance, saving
             plt.subplot(8, 5, i + 1)
         if instrument == 'JWST':
             plt.subplot(6, 3, i + 1)
+        if instrument == 'RST':
+            plt.subplot(6, 3, i + 1)
```

Comment:
@davidbourgeois I know now what you mean with the error you get. These subplots are supposed to plot each pastis mode into a grid, which means that you need to have at least as many grid spots like modes. For RST, you have 48x48=2304 modes, but here you are only using 6x3=18 grid points.

Since the sheer amount of modes with the continuous mirrors is not really feasible to put into a single plot, we might want to skip this figure altogether when working on a continuous mirror. But we'll look at this later, when we start working on on the analysis.